moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -114,6 +115,11 @@ ServoParameters::SharedConstPtr ServoParameters::makeServoParameters(const rclcp
   declareOrGetParam<std::string>(parameters->move_group_name, ns + ".move_group_name", node, logger);
   declareOrGetParam<std::string>(parameters->planning_frame, ns + ".planning_frame", node, logger);
   declareOrGetParam<std::string>(parameters->ee_frame_name, ns + ".ee_frame_name", node, logger);
+  declareOrGetParam<bool>(parameters->is_primary_planning_scene_monitor, ns + ".is_primary_planning_scene_monitor",
+                          node, logger, true);
```

Comment:
I think this should default to the same behavior we had previously. So, shouldn't this default to `false`?

Reply:
@JafarAbdi can you answer this question? Then I'm good to merge this

Reply:
I think defaulting to true is reasonable because if you default to false you'll need to change it to true for every case where you want to run servo by itself without another psm running.